src/app/pro/templates/page.tsx

```
@@ -0,0 +1,303 @@
+          {templates.map((template) => (
+            <TemplateCard key={template.id} template={template} />
+          ))}
+        </div>
+      </div>
+
+      {/* Coming Soon Notice */}
+      <Card className="border-dashed">
+        <CardContent className="p-6 text-center">
+          <h3 className="font-semibold mb-2">More Templates Coming Soon</h3>
+          <p className="text-sm text-muted-foreground mb-4">
+            We're working hard to bring you more high-quality templates. 
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)